dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/utils/JacksonSerializer.java

```diff
@@ -42,7 +56,14 @@
 	 * @return string
 	 */
 	public static <T> String serializeToString(T obj)  {
-		return JSON.toJSONString(obj);
+		String json = "";
+		try {
```

Comment:
In original logic ,if obj can't format to json ,exceptions throw. However current method return empty string ,which maybe change the logic of the invoker